Production_SOP.md

```diff
@@ -0,0 +1,71 @@
+# Core Operations Production SOP
+## UC Santa Cruz Genomics Institute, Computational Genomics Lab
+ 
+## Environment Setup
+ 
+ These steps should only have to be run once. Toil-box images will have to be updated periodically to stay current.
+  
+### Setup CGCloud
+Comprehensive help is found at: https://github.com/BD2KGenomics/cgcloud .
+ The following steps are provided for convenience but it is recommended to read the source documentation.
+
+1. `virtualenv ~/cgcloud`
+2. `source cgcloud/bin/activate`
+3. `pip install cgcloud-toil==1.5.5`
+4. `deactivate`
+5. `ln -snf ~/cgcloud/bin/activate ~/bin/`
+6. Set `~/.profile` options from CGCloud README
```

Comment:
Be explicit. No linking or references except for optional steps. This document should be self contained. Be sure to mention `CGCLOUD_KEYPAIRS`.